src/core/defaults.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Sets the core defaults
+ */
+export const name = "core/defaults";
+import { rule as checkInternalSlots } from "./linter-rules/check-internal-slots";
+import { rule as checkPunctuation } from "./linter-rules/check-punctuation";
+import { definitionMap } from "./dfn-map";
```

Comment:
Only import this one in the `w3c/defaults` (see the comment in that file). Please also remove the where definitionMap is assigned to conf.